Formula/transmission-cli.rb

```diff
@@ -1,16 +1,12 @@
 class TransmissionCli < Formula
   desc "Lightweight BitTorrent client"
   homepage "https://www.transmissionbt.com/"
-  url "https://github.com/transmission/transmission-releases/raw/d5ccf14/transmission-3.00.tar.xz"
-  sha256 "9144652fe742f7f7dd6657716e378da60b751aaeda8bef8344b3eefc4db255f2"
+  # pull from git tag to get submodules
```

Comment:
What submodules are needed?

Reply:
https://github.com/transmission/transmission/blob/main/.gitmodules
Some are not available on Homebrew (e.g., [ckormanyos/wide-integer](https://github.com/ckormanyos/wide-integer)).

Reply:
They should be added then, rather than just duplicating a bunch of Homebrew formula

Reply:
If we need to recursively include a bunch of specific Git submodules in home brew as individual formula (and somehow handle versioning, as I expect none of them are at HEAD) then I’m not sure I have time for that - this has already taken more than I expected and I don’t even use transmission.

If someone else wants to pick this up then please feel free.

Reply:
Because https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae

> Homebrew formulae should avoid having multiple, separate, upstream projects bundled together in a single package to avoid shipping outdated/insecure versions of software that is already a formula. Veracode’s [State of Software Security report](https://www.veracode.com/blog/research/announcing-state-software-security-v11-open-source-edition) concludes:

> In fact, 79% of the time, developers never update third-party libraries after including them in a codebase.

> For more info see [Debian’s](https://www.debian.org/doc/debian-policy/ch-source.html#s-embeddedfiles) and [Fedora’s](https://docs.fedoraproject.org/en-US/packaging-guidelines/#bundling) stances on this.

Reply:
I agree what they are doing is dumb and the rationale behind the rule, but how do you handle the extra patches they have applied to some dependencies?

it also seems the submodules are all-or-nothing, which is going to be a huge time sink.